cmake/scripts/webos/Install.cmake

```diff
@@ -1,4 +1,8 @@
 # webOS packaging
+find_program(ARES_PACKAGE ares-package PATHS ${TOOLCHAIN}/../CLI ENV WEBOS_CLI_PATH
```

Comment:
Might be better to use HINTS instead of PATHS?

HINTS is a higher search preference, PATHS is final resort. Generally if we expect it to be found in our given path, we use HINTS, as theres potential sys env var paths can be searched before PATHS are searched. Realistically it probably wont ever matter, and i know cmake say HINTS should be introspected paths found, rather than a hard coded path, so go with whatever. 

More a query than anything else.

Reply:
yeah, we expect it to be found in PATH, so this is more of a fallback and in line with the expected search order.